crates/fda/src/cli.rs

```diff
@@ -272,6 +272,18 @@ pub enum PipelineAction {
         #[arg(value_hint = ValueHint::Other, add = ArgValueCompleter::new(pipeline_names))]
         name: String,
     },
+    /// Retrieve the logs of a pipeline.
+    #[clap(aliases = &["log"])]
+    Logs {
+        /// The name of the pipeline.
+        #[arg(value_hint = ValueHint::Other, add = ArgValueCompleter::new(pipeline_names))]
+        name: String,
+        /// Watch the log endpoint and emit new pipeline log messages as they are written.
```

Comment:
this information should be somewhere in the documentation too.
is the rustdoc enough?

Reply:
`fda log --help` will display this

Reply:
can you a README file which is automatically generated from fda --help?

Reply:
we have this: https://docs.feldera.com/reference/cli I can add the logs command as an example